lib/ProductOpener/Products.pm

```diff
@@ -304,17 +341,13 @@ e.g. off:[code]
 
 =head3 Parameters
 
-=head4 Owner id
+$Owner id & $code [Product barcode]
```

Comment:
What is the reason for the removal of the =head4 header for each parameter?

Reply:
@stephanegigandet: I could be wrong but from what I have seen the documentation format is like- 
=`head_N` "the header"
"the content for the heading"

so, 
= `head3` Parameters
`$Parameter1 Parameter2 ` (the parameters)

Reply:
so the idea in the current doc is to have a header3 for the Parameters, and then header4 for each parameter, to separate them.

e.g. you can see how it looks when changed to HTML: (the file is old, I need to regenerate it)

https://world.openfoodfacts.org/files/doc/perl/ProductOpener/Ecoscore.html

In markdown, it would be like this:

### Parameters

#### First parameter

The first parameter is blabla

#### Second parameter

...

Reply:
thank you, fixed it!